tests/pinkopy/test_base_session.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+from pinkopy.base_session import BaseSession
+from tests.pinkopy import test_helper
+
+
+class TestPinkopyError(unittest.TestCase):
+    pass
+
+
+class TestBaseSessionMethods(unittest.TestCase):
+    def test__init__(self):
+        expected = test_helper.mock_session(BaseSession)
+        base_session = expected['Session']
+        test_helper.validate_base_session(expected, base_session)
+
+    def test__enable_method_cache(self):
+        base_session = test_helper.mock_session(BaseSession)['Session']
+        for method_name in base_session.cache_methods:
+            method = getattr(base_session, method_name)
+            assert method.ttl_cache() is not None
```

Comment:
I did this based on lines 65-66 in base_session:

```
if not inspect.isfunction(method.cache_info):
                setattr(self, method_name, ttl_cache(ttl=self.cache_ttl)(method))
```

I could be misunderstanding this though